parser-typechecker/src/Unison/Codebase/Branch.hs

```diff
@@ -113,6 +113,20 @@ makeLenses ''Branch
 makeLensesFor [("_edits", "edits")] ''Branch0
 makeLenses ''Raw
 
+-- | Like @deepTerms@, but returns relative names instead.
+--
+-- FIXME This should replace @deepTerms@ eventually, since they are all
+-- relative.
+relDeepTerms :: Branch0 m -> Star Referent RelName
```

Comment:
This and `relDeepTypes` are just a couple examples of moving towards more type safety.